targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala

```diff
@@ -82,7 +81,7 @@ object MagnitudeForm {
               )
             )
 
-          Form(size = Small)(
+          React.Fragment(
```

Comment:
Why this change? Seems unrelated

Reply:
Yes, it's unrelated. There was a `Form` inside of another `Form` (from `TargetBody`).